### DISCUSSION TITLE
how to change the video metadata transfer from hlg to pq?

### DISCUSSION THREAD

**Discussion by geextahslex** [mpv-player/mpv#15504] [Q&A] [answered]
Hi, how to change the video metadata transfer from hlg to pq?

Thank you :)

**Comment by kasper93** [comment, marked as answer]
`--vf=format=gamma=pq`

**Reply by geextahslex** [reply]
Thank you but it still shows `hlg`
![hlg22](https://github.com/user-attachments/assets/17b2c159-a721-48f4-8277-5b917a63af36)

**Reply by kasper93** [reply]
there was a typo, try now.

**Reply by geextahslex** [reply]
lol I also missed this ^^ g and q 
Thank you, that worked :D
